core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -76,6 +79,28 @@
    * <code>avro.compatibility.level</code>
    */
   public static final String COMPATIBILITY_CONFIG = "avro.compatibility.level";
+  public static final String KAFKASTORE_SECURITY_PROTOCOL_CONFIG =
```

Comment:
This change would require we use Kafka's `ConfigDef`, which we decided we wouldn't do now. So although it's a great idea, for now we can't do it.